code/modules/jobs/job_types/botanist.dm

```diff
@@ -9,6 +9,9 @@
 	supervisors = "the head of personnel"
 	selection_color = "#bbe291"
 	chat_color = "#95DE85"
+	minimal_player_age = 1
+	exp_requirements = 120
+	exp_type = EXP_TYPE_CREW
```

Comment:
Botanist is a pretty new player friendly role, even if you don't know genetic mutations with it you can still grow normal plants.

Reply:
This one isn't because new players - it's because botany is a low-hanging fruit for fresh account griefers primarily.

Reply:
> This one isn't because new players - it's because botany is a low-hanging fruit for fresh account griefers primarily

I don't think that should be the main reasoning behind applying playtime restrictions. It's pretty easy to catch potential botany griefers in the act compared to say, a plasma flooder or bomb maker

Reply:
I disagree quite a bit - there's no simple way to watch for someone to be preparing grief with botany because that's what normal botanists look like they're doing 90% of the time strange seeds get involved.

On the contrary, it's very easy to spot someone pumping plasma into distro or prepping maxcaps and then leaving science with them.